<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>codecServer</class>
 <widget class="QWidget" name="codecServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>codecServer</string>
  </property>
  <widget class="QTreeView" name="devTreeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PORT ï¼š</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortLineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>8001</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serverStartButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ServerStart</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>491</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="log">
    <attribute name="title">
     <string>Log</string>
    </attribute>
    <widget class="QTextBrowser" name="logTextBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="devInfo">
    <attribute name="title">
     <string>DevInfo</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="IpLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IP :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IpLineEdit">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
